components/EmptyListView.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import { connectStyle } from '@shoutem/theme';
+import { Text, View } from '@shoutem/ui';
+import EmptyList from '../assets/images/emptyList.svg'
+
+class EmptyListView extends PureComponent {
+  static defaultProps = {
+    title: "It's empty in here",
+    message: "We couldn't find anything to show...",
+  }
+
+  render() {
+    const { message, title, style } = this.props
+
+    return (
+      <View styleName="vertical h-center">
+        <EmptyList style={style.image} />
```

Comment:
I'd prefer dynamic values for this. I may not want chat type of image for every list

Reply:
Also, it would be cool to support animation here. more of a design task, but maybe Poriluk can hook us up

Reply:
Agree